CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+##Contributing
```

Comment:
Hey @lukebakken - given the common approach of having a CONTRIBUTING.md file in a repo these days, I'd prefer to delete the page on the wiki and leave it in the repo for this one case. All other docs can of course go in the wiki. Would that work ?